src/components/EmojiPicker/EmojiPickerMenuItem/index.native.tsx

```diff
@@ -0,0 +1,49 @@
+import React, {useEffect, useRef} from 'react';
+import PressableWithoutFeedback from '@components/Pressable/PressableWithoutFeedback';
+import Text from '@components/Text';
+import useStyleUtils from '@hooks/useStyleUtils';
+import useThemeStyles from '@hooks/useThemeStyles';
+import getButtonState from '@libs/getButtonState';
+import CONST from '@src/CONST';
+import type EmojiPickerMenuItemProps from './types';
+
+function EmojiPickerMenuItem({emoji, onPress, onHoverIn, onHoverOut, onFocus, onBlur, isFocused, isHighlighted, isUsingKeyboardMovement}: EmojiPickerMenuItemProps) {
+    const ref = useRef<HTMLDivElement | null>(null);
+    const StyleUtils = useStyleUtils();
+    const themeStyles = useThemeStyles();
+
+    useEffect(() => {
+        if (!isFocused) {
+            return;
+        }
+
+        ref?.current?.focus();
+    }, [isFocused]);
+
+    return (
+        <PressableWithoutFeedback
+            shouldUseAutoHitSlop={false}
+            onPress={() => onPress(emoji)}
+            onHoverIn={onHoverIn}
+            onHoverOut={onHoverOut}
+            onFocus={onFocus}
+            onBlur={onBlur}
+            ref={ref}
+            style={({pressed}) => [
+                StyleUtils.getButtonBackgroundColorStyle(getButtonState(false, pressed)),
+                isHighlighted && isUsingKeyboardMovement ? themeStyles.emojiItemKeyboardHighlighted : {},
+                isHighlighted && !isUsingKeyboardMovement ? themeStyles.emojiItemHighlighted : {},
```

Comment:
This should work as well:
```suggestion
                isHighlighted && isUsingKeyboardMovement && themeStyles.emojiItemKeyboardHighlighted,
                isHighlighted && !isUsingKeyboardMovement && themeStyles.emojiItemHighlighted,
```